source/Ship.cpp

```diff
@@ -3469,17 +3469,26 @@ double Ship::Mass() const
 
 
 
+// Account for inertia reduction, which only affects the ship's mass and not cargo or carried ships.
+// It also has no effect on maximum heat or heat capacity.
+double Ship::InertialMass() const
+{
+	return carriedMass + cargo.Used() + attributes.Mass() / (1 + attributes.Get("inertia reduction"));
```

Comment:
```suggestion
	return carriedMass + cargo.Used() + attributes.Mass() / (1. + attributes.Get("inertia reduction"));
```
Isnt cargo already accounted for by carried mass? What about Mass()? Also mb we could cache this but its not big deal if we dont
Btw shouldnt there be a parenthesis?

Reply:
I still want my . then x)